runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -1932,6 +1960,18 @@ TR::ILOpCodes TR_VectorAPIExpansion::ILOpcodeFromVectorAPIOpcode(int32_t vectorA
             return TR::BadILOp;
          }
       }
+   else if (opCodeType == MaskReduction)
+      {
+      switch (vectorAPIOpCode)
+         {
+         case VECTOR_OP_MASK_TRUECOUNT: return scalar ? TR::BadILOp : TR::ILOpCode::createVectorOpCode(TR::mTrueCount, vectorType);
+         case VECTOR_OP_MASK_FIRSTTRUE: return scalar ? TR::BadILOp : TR::ILOpCode::createVectorOpCode(TR::mFirstTrue, vectorType);
+         case VECTOR_OP_MASK_LASTTRUE:  return scalar ? TR::BadILOp : TR::ILOpCode::createVectorOpCode(TR::mLastTrue,  vectorType);
+         case VECTOR_OP_MASK_TOLONG:    return scalar ? TR::BadILOp : TR::ILOpCode::createVectorOpCode(TR::mToLongBits,    vectorType);
```

Comment:
Please adjust the number of spaces before `vectorType`.

Reply:
Done. Also added minor improvement to tracing.